_content/tour/por/constants.article

```diff
@@ -1,123 +1,121 @@
-Constants
-One of the more unique features of Go is how the language implements constants.
+Constantes
+Uma característica distintiva da linguagem Go é como a linguagem implementa constantes.
 
-* Constants
+* Constantes
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Assista ao vídeo]]
+- Precisa de Assistência Financeira, Use Nosso [[https://www.ardanlabs.com/scholarship/][Formulário de Bolsa de Estudos]]
 
-One of the more unique features of Go is how the language implements constants.
-The rules for constants in the language specification are unique to Go. They provide
-the flexibility Go needs to make the code we write readable and intuitive while still
-maintaining type safety.
+Uma das características mais singulares de Go é a forma como a linguagem implementa constantes.
+As regras para constantes na especificação da linguagem são exclusivas para Go. Elas oferecem
+a flexibilidade que Go precisa para tornar o código que escrevemos legível e intuitivo, enquanto ainda
+mantém a segurança de tipo.
 
-Constants can be typed or untyped. When a constant is untyped, it’s considered to
-be of a kind. Constants of a kind can be implicitly converted by the compiler. This
-all happens at compile time and not at runtime.
+Constantes podem ser tipadas ou não tipadas. Quando uma constante é não tipada, ela é considerada
+de um certo "tipo" (em inglês "kind"). Constantes desse "tipo" podem ser convertidas implicitamente pelo compilador. 
```

Comment:
I think we don't have a good translation for kind, in my opinion we should not translate. It's very hard to distinguish when reading and impossible when speaking.